generator/sbpg/targets/resources/sbp_messages_template.rs

```diff
@@ -35,11 +34,12 @@ use super::(((i)))::*;
 (((m.desc|commentify)))
 ///
 ((*- endif *))
-#[cfg_attr(feature = "sbp_serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "sbp_serde", derive(serde::Serialize))]
 #[derive(Debug, Clone)]
 #[allow(non_snake_case)]
 pub struct (((m.identifier|camel_case))) {
     ((*- if m.is_real_message *))
+    #[cfg_attr(feature = "sbp_serde", serde(skip_serializing))]
```

Comment:
Skip the `sender_id` because the json encoders add it as apart of the `common fields` added to all messages